library/lua/3rdparty/alt_getopt.lua

```diff
@@ -45,6 +45,9 @@
 
 local _ENV = mkmodule('3rdparty.alt_getopt')
 
+---@nodiscard
+---@param opts string
+---@return table<string, string|integer>
```

Comment:
table value is always an integer (but I see that the more permissive type might be needed because it can be string|integer in other contexts)

Reply:
Yeah this is one of a few "just has to be truthy" fields.